layouts/resources/list.html

```diff
@@ -6,17 +6,14 @@
 <div class="relative max-w-7xl mx-auto px-4 sm:px-2">
     <section class="mt-9 sm:mt-11 lg:mt-14">
         <ul class="grid md:grid-cols-2 xl:grid-cols-3 gap-6 md:gap-8">
-            {{ $paginator := .Paginate .Data.Pages }}
```

Comment:
I'd like to test not paginating this page for now since it has less content than the blog page and users may not be as interested in clicking through all of the resources.

Reply:
I've been thinking about reducing the amount of clicks a user has to make as some may prefer to view everything on 1 page. Besides that, I don't think there's any real downside.